### ISSUE TITLE
Buttercup csv file is not recognized

### ISSUE THREAD

**Issue by JuliusHerrmann** [roddhjav/pass-import#102]
Hi,
Sorry if I'm making some obvious mistakes.
Anyway, I exported my buttercup passwords to a file put wehn i run pass
`import buttercup filename.csv`
The output is:
`[x] Error: test is not a valid exported buttercup file.`
I also tried to use the default csv option but I just could not figure it out.
I also realised that the csv file tries to sort items with , (comma) but some of my passwords also include this letter. I suppose this could mess up the parsing?
~ Thanks ~

**Comment by roddhjav** [comment]
Hi,

You can check the format of your CSV file by comparing it against the [test file](https://github.com/roddhjav/pass-import/blob/master/tests/assets/db/buttercup.csv). If there are not the same, then you should provide me with your CSV header (and buttercup version)

To have more information regarding the error, you can also run: `pass import buttercup filename.csv -vvv`

> I also realised that the csv file tries to sort items with , (comma) but some of my passwords also include this letter. I suppose this could mess up the parsing?

To respect the CSV format, the program that generated the file is supposed to quote value when needed. So it *should* not be an issue.

**Comment by JuliusHerrmann** [comment]
Alright,
First of all thank you!
I managed to find the issue: I did not have any notes in my buttercup entries. Buttercup then exported the csv file without the column Notes in the header. Once I added it to the header it worked.
Also: It seems that buttercup handles custom labels as a new column. Meaning that a custom label would change the header in return messing up the expected structure?
As an example I have added the field MyCustomField to one of the entries and this is my header:
`!group_id,!group_name,title,username,password,URL,MyCustomField,!group_id,!group_name,id
`
I also did not have any entries with the label Notes in this example.
I was however still able to import these files once i added the column Notes to the header.
~ Thanks, I hope this helped in some way ~
